integrations/vello_svg/src/lib.rs

```diff
@@ -197,20 +229,21 @@ fn paint_to_brush(paint: &usvg::Paint, opacity: usvg::Opacity) -> Option<(Brush,
                     cstop.color.g = stop.color.green;
                     cstop.color.b = stop.color.blue;
                     cstop.color.a = (stop.opacity * opacity).to_u8();
-                    cstop.offset = stop.offset.get() as f32;
+                    cstop.offset = stop.offset.get();
                     cstop
                 })
                 .collect();
-            let start: vello::kurbo::Point = (gr.x1, gr.y1).into();
-            let end: vello::kurbo::Point = (gr.x2, gr.y2).into();
-            let transform = Affine::new([
-                gr.transform.a,
-                gr.transform.b,
-                gr.transform.c,
-                gr.transform.d,
-                gr.transform.e,
-                gr.transform.f,
-            ]);
+            let start: vello::kurbo::Point = (gr.x1 as f64, gr.y1 as f64).into();
+            let end: vello::kurbo::Point = (gr.x2 as f64, gr.y2 as f64).into();
+            let (a, b, c, d, e, f) = (
```

Comment:
Is this exactly the same code as before, but in a different style, or is it doing something different?

I'd expect both instances to be factored out if they're the same thing, although I see that the repetition is pre-existing

The new need to use `as f64` complicates things

Reply:
First, usvg switched to f32, reasons here:
https://github.com/RazrFalcon/resvg/blob/master/CHANGELOG.md#0340---2023-05-27

It is the same code as before, and was hard to get right. When usvg changed to [sx, kx, ky,...] instead of [a,b,c,d,e,f], it confused things for me. Fortunately I found this remapping helpful: https://github.com/RazrFalcon/resvg/blob/c6fd7f486aaef81704546f298541513f50f4f868/crates/c-api/lib.rs#L758

Usvg also uses column-major-column-vector matrix notation, therefore it’s ky-kx, not kx-ky.